common/main/en.xml

```diff
@@ -9305,130 +9305,130 @@ annotations.
 		<initialPattern type="initial">{0}.</initialPattern>
 		<initialPattern type="initialSequence">{0} {1}</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{title} {given} {given2} {surname} {generation}, {credentials}</namePattern>
+			<namePattern draft="provisional">{title} {given} {given2} {surname} {generation}, {credentials}</namePattern>
 		</personName>
```

Comment:
We don't want to reset _`en`_ to provisional, do we??

Reply:
Right, realized that I'd not excluded en.xml part-way through the meeting!

Reply:
Fixed that and a couple of other issues.